Hearthstone Deck Tracker/DeckManager.cs

```diff
@@ -537,10 +539,11 @@ public static void UpdateDungeonRunDeck(DungeonInfo info)
 					playerClass = ((CardClass)info.HeroCardClass).ToString().ToUpperInvariant();
 			}
 
-			var deck = DeckList.Instance.Decks.FirstOrDefault(x => x.IsDungeonDeck && x.Class.ToUpperInvariant() == playerClass.ToUpperInvariant()
-																		&& !(x.IsDungeonRunCompleted ?? false)
+			var deck = DeckList.Instance.Decks.FirstOrDefault(x => (!isPVPDR && x.IsDungeonDeck || isPVPDR && x.IsDuelsDeck) && x.Class.ToUpperInvariant() == playerClass.ToUpperInvariant()
+																		&& !((x.IsDungeonRunCompleted ?? false) || (x.IsDuelsRunCompleted ?? false))
```

Comment:
I'm not entirely sure whether this is just kind of hard to grap or actually wrong. But I think it's easier to understand if written as
```
&& !(x.IsDungeonRunCompleted ?? false)
&& !(x.IsDuelsRunCompleted ?? false)
```